packages/opentelemetry-exporter-prometheus/src/PrometheusSerializer.ts

```diff
@@ -63,6 +63,28 @@ function sanitizePrometheusMetricName(name: string): string {
   return name.replace(invalidCharacterRegex, '_'); // replace all invalid characters with '_'
 }
 
+/**
+ * @private
+ *
+ * Helper method which assists in enforcing the naming conventions for metric
+ * names in Prometheus
+ * @param name the name of the metric
+ * @param kind the kind of metric
+ * @returns string
+ */
+function enforcePrometheusNamingConvention(
+  name: string,
+  kind: MetricKind
+): string {
+  // If the metric kind is Counter if so ensure that the name has the suffix
```

Comment:
I would add here the comment from line 188. And at the same time would remove it from line 188 and 212. It will provide better understanding what is really happening here and why

```suggestion
 // Prometheus requires that metrics of the Counter kind have "_total" suffix
```